index.html

```diff
@@ -1,15 +1,140 @@
+            <h2>Main program</h2>
+            <hr>
+            <div class="cards">
+                <div class="card">
+                    <img src="./public/images/program_icon_01.png" alt="Lecturer icon">
+                    <h2>Lecture</h2>
+                    <p>Listen to the speakers from various countries about the messages of sharing and opening.
+                    </p>
+                </div>
+                <div class="card">
+                    <img src="./public/images/program_icon_02.png" alt="HTS exhibition icon">
+                    <h2>HTS Exhibition</h2>
+                    <p>Meet the creations of artists from various fields who share the open spirit of HTS.</p>
+                </div>
+                <div class="card">
+                    <img src="./public/images/program_icon_03.png" alt="Forum icon">
+                    <h2>Forum</h2>
+                    <p>We have time to share our thoughts and opinions with experts for each topic covered.</p>
+                </div>
+                <div class="card">
+                    <img src="./public/images/program_icon_04.png" alt="Workshop icon">
+                    <h2>Workshop</h2>
+                    <p>Instead of just looking at creations using open source, you can create them yourself.</p>
+                </div>
+                <div class="card">
+                    <img src="./public/images/program_icon_05.png" alt="HTS ignite icon">
+                    <h2>HTS Ignite</h2>
+                    <p>Create opportunities to freely network with HTS personnel from around the world.</p>
+                </div>
+            </div>
+            <a href="">see the whole program</a>
+        </section>
+
+        <section class="speakers">
+            <h2>Featured Speakers</h2>
+            <hr>
+            <div class="cards" id="featured-section"></div>
+        </section>
+
+        <section class="partner">
+            <h2>Partner</h2>
+            <hr>
+            <div class="cards">
+                <img src="./public/images/logo_airbnb.png" alt="Logo Airbnb">
+                <img src="./public/images/logo_nestle.png" alt="Logo Nestle">
+                <img src="./public/images/logo_mozilla.png" alt="Logo Mozilla">
+                <img src="./public/images/logo_blender.png" alt="Logo Bendler">
+            </div>
+        </section>
+
+        <section class="footer">
+            <div class="cards">
+                <div class="cc-logo">
+                    <img src="./public/images/creative_commons.png" alt="Logo Creative Commons">
+                </div>
+                <div class="legacy">
+                    <p><strong>2023 Creative Commons Haiti. Some Rights Reserved</strong></p></br>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quo necessitatibus deleniti nobis modi
+                        ab facere sequi iure omnis aspernatur!</p>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] You have done great working on this footer section as suggested by the previous reviewers. :+1: But I think there is a responsiveness issue you need to fix before moving to the next milestone, Although your logo looks good on desktop it does not resize correctly on mobile devices, the logo tends to overflow the page. Please you need to fix this it will be great if your project looks good in all major screen sizes. :+1: 

SCREENSHOT FOR REFERNCE:
![Selection_021](https://user-images.githubusercontent.com/107632780/235008990-68173da3-47e9-4a68-8e9a-801b06428303.png)